### PR TITLE
Portfolio draft

### PR DESCRIPTION
**Pull request by difanyang**
- Created several sections, including About, Projects, and Image Gallery
- Used a horizontal navigation bar to link between pages
- Added a background image
- Linked LinkedIn and GitHub profile pages

### CODE REVIEW

**Comment by floatfeather** [comment]
Could you please attach a screenshot of your UI?

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -13,16 +13,18 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random fact about myself to the page.
```
Comment (zixiyan) [inline portfolio/src/main/webapp/script.js:16]: Consider comment this function like this: https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -1,9 +1,65 @@
...
+    padding: 0;
+    overflow: hidden;
+    font-family: "Courier New", Courier, monospace;
+}
+
+#navBar li {
+    float: left;
+}
+
+#navBar li a{
```
Comment (zixiyan) [inline portfolio/src/main/webapp/style.css:50]: why not a new space before "{"?
e.g.
#navBar li a {

**Comment by difanyang** [comment]
> Could you please attach a screenshot of your UI?

image
image
image
image

Please let me know if you cannot see these screenshots. Thanks!

**Pull request merged**